packages/client/mutations/toasts/mapTeamsLimitReminderToToast.ts

```diff
@@ -23,7 +24,7 @@ const mapTeamsLimitReminderToToast = (
 
   return {
     autoDismiss: 0,
-    key: `newNotification:${notificationId}`,
+    key: makeNotificationToastKey(notificationId),
```

Comment:
`notificationId` is a unique value, lets unify the `key` of toasts that mirrors specific notifications. This way we can remove them easily